oneanddone/tasks/migrations/0023_auto__add_field_feedback_how_many_minutes_did_you_spend_on_the_task.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
```

Comment:
This one needs to be prefixed with `0024` as I mentioned in my last comment. The PR that I just merged added migrations for 0022 and 0023.